sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/samplelist/ProjectSamplesView.groovy

```diff
@@ -0,0 +1,99 @@
+package life.qbic.portal.sampletracking.components.projectoverview.samplelist
+
+
+import com.vaadin.data.provider.ListDataProvider
+import com.vaadin.shared.ui.grid.HeightMode
+import com.vaadin.ui.Grid
+import com.vaadin.ui.VerticalLayout
+import life.qbic.datamodel.samples.Status
+import life.qbic.portal.sampletracking.communication.notification.NotificationService
+
+/**
+ * <b>A simple view containing a list of samples</b>
+ *
+ * @since 1.0.0
+ */
+class ProjectSamplesView extends VerticalLayout{
+    private final ViewModel viewModel
+    private final Presenter presenter
+    private Grid<Sample> samplesGrid
+
+    ProjectSamplesView(NotificationService notificationService) {
+        this.viewModel = new ViewModel()
+        this.presenter = new Presenter(notificationService, viewModel)
+        initLayout()
+    }
+
+    private void initLayout() {
+        this.setMargin(false)
+        this.samplesGrid = createSamplesGrid(viewModel.samples)
+        this.addComponents(samplesGrid)
+    }
+
+    Presenter getPresenter() {
+        return presenter
+    }
+
+    private static Grid<Sample> createSamplesGrid(Collection<Sample> samples) {
+        Grid<Sample> samplesGrid = new Grid<>()
+        ListDataProvider<Sample> dataProvider = ListDataProvider.ofCollection(samples)
+        samplesGrid.addColumn(Sample::getCode).setCaption("Sample Code").setId("SampleCode")
+        samplesGrid.addColumn(Sample::getTitle).setCaption("Sample Title").setId("SampleTitle")
+        samplesGrid.addColumn(Sample::getStatus).setCaption("Sample Status").setId("SampleStatus")
+        samplesGrid.setSelectionMode(Grid.SelectionMode.NONE)
+        samplesGrid.setDataProvider(dataProvider)
+        samplesGrid.setHeightMode(HeightMode.ROW)
+        samplesGrid.setSizeFull()
+        return samplesGrid
+    }
+
+    /**
+     * Bean for the Grid to be displayed
+     */
+    public static class Sample {
+        String code
+        String title
+        Status status
+
+        Sample(String code, String title, Status status) {
+            this.code = code
+            this.title = title
+            this.status = status
+        }
+    }
+
```

Comment:
why not using the sample DTO that has been introduced? I thought that was the whole idea of it. So you can transfer samples from the use case to the view.

Reply:
Please name your default branch like the story card we have for it so that it can be found. I had no idea there was a feature branch already. If you are ok with it I will rename it accordingly and change the base of this PR

Reply:
this did not answere my question here 🙈 what about the DTO?

Reply:
I used it now :D That is why I resolved this comment. It was not there when I originally opened the PR